### ISSUE TITLE
False positive for contrast ratio

### ISSUE THREAD

**Issue by mfairchild365** [squizlabs/HTML_CodeSniffer#89]
I recently got this message:

```
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.5:1. Recommendation: change background to #00892c.
```

Foreground color used: #FFFFFF
background color used: #008A2C

I found this to be confusing because the error message said that it expected a ratio of `4.5:1` but it found a ratio of `4.5:1`.  We went ahead and used the suggested color, but from the error text, it sounds like it shouldn't have been an error...

Also, this service says that the ratio was valid: http://snook.ca/technical/colour_contrast/colour.html

Not sure who to believe.
